src/lib/gossip.ts

```diff
@@ -153,7 +147,8 @@ export class Gossip {
 
   readyToDive(): boolean {
     return (
-      this.stench >= BASE_STENCH_REQUIRED + this.players.length + this.requestingCompost.length
+      parseInt(get(CURRENT_STENCH)) >=
+      BASE_STENCH_REQUIRED + Math.ceil(this.players.length * 1.1) + this.requestingCompost.length
```

Comment:
Do we need the wiggle room on `this.players`?

Reply:
it was a suggestion by icyman in their PR as insurance against any errors in tracking. I'm ok with removing this margin. Will say even with margin, we will be accumulating less stench than we are currently based on what I've seen. 

I slightly lean towards keeping it as I like margin, but fine with removing if you prefer. lmk

Reply:
I'm cool with this either way. Ah, one other point, the current implementation doesn't play nicely with composting.

Reply:
The current implementation removes stench as folks compost. So each successful compost will cause this condition to be falsey.

Reply:
I did it this way because it is more accurate. But you might need to change some conditions that currently say "8 + players + compost" to "8 + players".
The reason why I like it better this way is that composting might cause the check stench at 30 dive attempts to be inaccurate, since stench damage got lowered but not that is not reflected in the stench level variable.

Reply:
Don't have time to look closer at the compost code, so apologies for a rushed response. I like composting lowering stench as that matches reality. We don't want to be in a situation where there isn't enough stench for someone to dive. Perhaps care is correct in that number of players requesting to compost is removed from the calculation. We are erring on the side of too much stench atm, which at worse is an efficiently hit. We are already going well above needed stench so I'd vote merge as is and refine in a future PR